.github/workflows/release.yml

```diff
@@ -56,28 +93,57 @@ jobs:
             path: "Refresher/bin/Release/net8.0-windows/win-x64/publish/"
             if-no-files-found: error
             retention-days: 1
+
+      - name: Upload macOS x64 build
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3.1.1
+        with:
+            name: "Refresher_for_macOS_x64"
+            path: "Refresher-x64.tar"
+            if-no-files-found: error
+            retention-days: 1
+  
+      - name: Upload macOS ARM64 build
+        if: matrix.os == 'macos-latest'
+        uses: actions/upload-artifact@v3.1.1
+        with:
+            name: "Refresher_for_macOS_ARM64"
+            path: "Refresher-arm64.tar"
+            if-no-files-found: error
+            retention-days: 1
   release:
     name: Release Built Artifacts
     needs: build
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
+        id: src
+
       - name: Download artifacts
         id: download
         uses: actions/download-artifact@v3.0.2
 
       - name: Create zip files
         run: |
             cd ${{steps.download.outputs.download-path}}/Refresher_for_Windows_x64
-            zip -r "${{steps.download.outputs.download-path}}/Refresh_for_Windows_x64.zip" .
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_Windows_x64.zip" .
             
             cd ${{steps.download.outputs.download-path}}/Refresher_for_Linux_x64
-            zip -r "${{steps.download.outputs.download-path}}/Refresh_for_Linux_x64.zip" .
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_Linux_x64.zip" .
+
+            cd ${{steps.download.outputs.download-path}}/Refresher_for_macOS_x64
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_macOS_x64.zip" .
+
+            cd ${{steps.download.outputs.download-path}}/Refresher_for_macOS_ARM64
+            zip -r "${{steps.download.outputs.download-path}}/Refresher_for_macOS_ARM64.zip" .
```

Comment:
Mac users probably know the names `Intel` and `Apple Silicon` better than `x64` and `arm64`

Reply:
I would say keeping this the way it is makes more sense, and instead adding a note in RELEASE_NOTES.md (as you requested) for "which architecture" makes more sense. This makes things easier for Linux and Windows arm64 builds as well since there are more brands than just one for both. (Linux and Windows arm64 was discussed in the Discord server, however it's not for this PR but instead another one.)

Reply:
I like the idea of consistency, but on the other hand, the term 'Apple Silicon' is pretty heavily used in Apple's marketing. Linux users know what they're doing and know they're on arm64. Microsoft doesn't really seem to have any naming other than Windows on Arm.

Apple, however, has their own name, so they don't really say their Macs are arm64, or even x86 for that matter. The average user is less likely to know arm64 than Apple Silicon. We can put a table, but I fear that users aren't going to read that. I don't see the point in adding a chart for something we can easily explain with a filename.

I've also seen other software referring to arm64 as Apple Silicon, for instance JetBrains calls it by Apple Silicon: https://intellij-support.jetbrains.com/hc/en-us/articles/360018972639-Apple-Silicon-Support-in-JetBrains-IDEs

I really think this naming should be front and center for Mac. We can adjust for other platforms as needed, but this is the state for Mac.

Reply:
Whatever you think works - although I think a look at the below comments would do well since it would remove the need to distinguish architectures at all.